region/norfair/west/Crocomire Speedway.json

```diff
@@ -669,6 +669,44 @@
         "FIXME: it can be possible to get through tankless without Wave, with enough arm pumping, or a cross-room runway."
       ]
     },
+    {
+      "link": [2, 4],
+      "name": "Use Flash Suit",
+      "requires": [
+        {"heatFrames": 970},
```

Comment:
- There needs to be tech for making it to the speedblocks, past the violas and pirates.
- Shinesparking from the leftmost speedblock should be the default case and everything else with Morph would require heatproof.